pkg/api/helpers.go

```diff
@@ -437,6 +437,12 @@ const (
 	// PreferAvoidPodsAnnotationKey represents the key of preferAvoidPods data (json serialized)
 	// in the Annotations of a Node.
 	PreferAvoidPodsAnnotationKey string = "scheduler.alpha.kubernetes.io/preferAvoidPods"
+
+	// SysctlsPodAnnotationKey represents the key of sysctls which are set for the infrastructure
+	// container of a pod. The annotation value is a comma separated list of sysctl_name=value
```

Comment:
In the proposal you mentioned `:` as the separator. You use `=` here. I'm ok with either. Just fix this PR or the proposal.